code/src/components/AddTask.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import uniqid from 'uniqid'
+import tasks from 'reducers/tasks'
+
+
+//Styled components
+const devices = {
+    mobile: "(min-width: 375px)",
+    tablet: "(min-width: 768px)",
+    desktop: "(min-width: 1025px)",
+}
```

Comment:
Oh, this was a clever way to "declare" the different viewports. I'm taking a little screen shot of this if you don't mind :)